hugo/layouts/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-Agent: *
+
```

Comment:
Extra blank space.
`robots.txt` use to use blank lines to separate groups [examples](https://developers.google.com/webmasters/control-crawl-index/docs/robots_txt#agrupacin-de-registros) instead of rules